doc/user-manual/language/polarity.lagda.rst

```diff
@@ -0,0 +1,109 @@
+..
+  ::
+  {-# OPTIONS --polarity #-}
+
+  module language.polarity where
+
+  open import Agda.Builtin.Nat
+
+.. _polarity:
+
+======================
+ Polarity Annotations
+======================
+
+Agda now supports explicitely annotating functions arguments and datatype
```

Comment:
In contrast to the changelog, the user manual should avoid talking about how Agda changed but just describe the current state of Agda. So you can remove the word "now" here.